SomosMasApp/Pods/Pods.xcodeproj/xcuserdata/marianobattaglia.xcuserdatad/xcschemes/Alamofire.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1240"
+   LastUpgradeVersion = "1330"
```

Comment:
this file should not be committed, any file on xcuserdata or that have your name in the FILE PATH